### DISCUSSION TITLE
Is it possible to delete a week from the schedule?

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#16] [Q&A] [answered]
I accidentally added a week :(

![Screenshot_14](https://user-images.githubusercontent.com/76891864/217392052-ee685633-2fbd-4219-9b84-4e95d84912b1.png)

**Comment by rhahao** [comment, marked as answer]
Hi @joaochris, 

Unortunately that is not possible yet. In fact, we believe that there is no reason why deleting is necessary. It will still be filled with the correct source when available.

Do you have any idea why allow deleting is necessary?

Thanks

**Reply by joaochris** [reply]
I understand! I saw that in the translation files a confirmation message appears, but it did not appear. I don't know if it was supposed to show up 😅. I think it's more a question of organization, of having only those weeks that have the source material at the moment. I think leaving them showing can become a little confusing, since there is no source material available 😃.

**Reply by rhahao** [reply]
Mmh... Now I see too that the message confirmation is not showing. That’s why the issue then. A message should display before it adds the week.

**Reply by rhahao** [reply]
I will create a work item to fix it. Thanks for reporting. 😂

**Reply by joaochris** [reply]
I saw that there is only one way to delete the week, it would be deleting everything. In this case, I have already registered all the students. But I wish I could save them to import them after deleting. Do you see an easier way to do this?

Sorry for my mistake.

**Reply by rhahao** [reply]
Unfortunately, deleting everything and importing again the backup (the only way to save your previous list for now) will not work. Let’s just hope that the delete feature will be available soon. 😊

**Comment by rhahao** [comment]
Hello @joaochris, this [PR #847](https://github.com/sws2apps/lmm-oa-sws/pull/847) now adds a possibility to delete week from the source materials list.

**Reply by joaochris** [reply]
Thanks!!
